CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace CryptoTechReminderSystem.Gateway
+{
+    public class HarvestGateway
```

Comment:
:eyes: Looks like you're missing an interface - Things will need to depend on the interface that this class inherits if they want to use it in the future

When working in a typed language like C# it's usually good to start by defining your interface, rather than the class first, as that's usually how you'd start to think about the system

Reply:
Yes, we have implemented the interface in bcb8cc131808ddebd5be51d4f1ef6e32fb3bd26a.